src/AppBundle/Controller/ShopController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace AppBundle\Controller;
+
+use AppBundle\Repository\AquariumRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+class ShopController extends Controller {
```

Comment:
There is no need to extend Symfony controller here. You don't use the underlining methods.